shared/settings/feedback.native.js

```diff
@@ -91,12 +92,8 @@ class Feedback extends Component<Props> {
             padding: globalMargins.tiny,
           }}
         >
-          <Icon
-            type="icon-fancy-feedback-96"
-            style={{height: 96, width: 96, alignSelf: 'center', marginBottom: globalMargins.tiny}}
-          />
-          <Text style={{textAlign: 'center'}} type="BodySemibold">
-            Please send us any feedback or describe any bugs you’ve encountered.
+          <Text style={{textAlign: 'center'}} type="HeaderBig">
+            {heading || 'Your feedback is welcomed!'}
```

Comment:
should just move 'your feedback is welcomed!' to the connector and set it if its not in route props